server.js

```diff
@@ -1,5 +1,69 @@
 // import your node modules
-
+const express = require('express')
+const bodyParser = require('body-parser')
 const db = require('./data/db.js');
 
 // add your server code starting here
+const server = express();
+server.use(bodyParser.json())
```

Comment:
Since we're using `express.json()` on the next line, we don't need `body-parser` any longer. They both do the same thing, parsing request bodies sent in JSON format. `express.json()` was included in Express 4.16 (https://expressjs.com/en/api.html#express.json)